### PR TITLE
close #48 - added hoverTextStyle property

### LINKED ISSUE #48: add hoverTextStyle in SideBarXTheme
**Describe the solution you'd like**
I would like a hoverTextStyle  property to be used in conjunction with hoverColor

.

### PR DESCRIPTION
**Pull request by MaurizioSodano**
I've added the hoverTextStyle to change the SidebarXCell textStyle on hover

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "git-blame.gitWebUrl": ""
+}
```
Comment (Frezyx) [inline .vscode/settings.json:3]: Can you remove this changes from pr?

File: lib/src/theme/sidebarx_theme.dart [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+      this.itemDecoration,
+      this.selectedItemDecoration,
+      this.itemMargin,
+      this.selectedItemMargin,
+      this.itemPadding,
+      this.selectedItemPadding,
+      this.itemTextPadding,
+      this.selectedItemTextPadding,
+      this.hoverColor,
+      this.hoverTextStyle});
```
Comment (Frezyx) [inline lib/src/theme/sidebarx_theme.dart:23]: Please add comma here

File: lib/src/theme/sidebarx_theme.dart [hunk 3]
```diff
@@ -153,28 +160,29 @@ class SidebarXTheme {
...
+            selectedItemDecoration ?? this.selectedItemDecoration,
+        itemMargin: itemMargin ?? this.itemMargin,
+        selectedItemMargin: selectedItemMargin ?? this.selectedItemMargin,
+        itemPadding: itemPadding ?? this.itemPadding,
+        selectedItemPadding: selectedItemPadding ?? this.selectedItemPadding,
+        itemTextPadding: itemTextPadding ?? this.itemTextPadding,
+        selectedItemTextPadding:
+            selectedItemTextPadding ?? this.selectedItemTextPadding,
+        hoverColor: hoverColor ?? this.hoverColor,
+        hoverTextStyle: hoverTextStyle ?? this.hoverTextStyle);
```
Comment (Frezyx) [inline lib/src/theme/sidebarx_theme.dart:186]: Please add comma here

**Review by Frezyx — CHANGES_REQUESTED** [review]

**Comment by Frezyx** [comment]
Hello @MaurizioSodano 
Thanks a lot for your contribution! 

Please fix review issues and after that run command:

 ```bash
 flutter format lib/
 ```

And I will merge it after changes

**Comment by MaurizioSodano** [comment]
Hello @Frezyx , happy to contribute.

I hope everything is fine with this update.
BR,
Maurizio

**Review by Frezyx — APPROVED** [review]

**Pull request merged**